.github/workflows/dd-build.yml

```diff
@@ -36,7 +36,8 @@ jobs:
       env:
         KUBE_BUILD_PLATFORMS: ${{ matrix.platform }}
         KUBE_RELEASE_RUN_TESTS: n
-      run: make quick-release CGO_ENABLED=1 KUBE_CGO_OVERRIDES="kube-apiserver kube-controller-manager kube-scheduler kubelet" KUBE_BUILD_PLATFORMS=$KUBE_BUILD_PLATFORMS
+        GOFLAGS: "-tags=fips"
```

Comment:
Nit better keep the alphabetical order in envvars

Reply:
fixed